app/assets/v2/js/pages/bounty_details.js

```diff
@@ -823,7 +823,7 @@ var build_detail_page = function(result) {
     });
   });
 
-  $('#bounty_details #issue_description code').parent().addClass('code-snippet');
+  $('#bounty_details #issue_description').parent().addClass('code-snippet');
```

Comment:
this would mean the entire issue description would be wrapped under grey box making it indistinguishable from the rest of the text right ? 
I assumed we'd look again at the plugin we've used to convert markdown (`.md`) to html and see if there is a better one out there

Reply:
Nope, this removes all the grey boxes from the markdown. What I suspect happens is that this piece of code finds all html tags with <code> in it, goes to its parent xml and adds class="code-snippet". This makes the grey box appear. 

![image](https://user-images.githubusercontent.com/9086733/54753770-d3ef9580-4c1c-11e9-9e48-0e25690e0c63.png)

I don't think the markdown is at fault here.

Reply:
I agree with @thelostone-mc. Firstly what you did right now, feels like it will add the `code-snippet` class to the entire issue-description. Now it might not appear grey, because by CSS, it is adding grey to `.code-snippet code`, but even then the logic is wrong. So in that case you could probably delete the entire line.

Also, it's not like we are not supposed to highlight or show the code section separately. This just looks weird and we hope to have a better way of showing.